### ISSUE TITLE
aria-live says only use assertive when "interruption is imperative". Why does aria-errormessage example use it rather than polite

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/aria#1139]
Aria specification  states
>  Because an interruption may disorient users or cause them to not complete their current task, authors SHOULD NOT use the assertive value unless the interruption is **_imperative_**.

 https://www.w3.org/TR/wai-aria-1.2/#aria-live

However the example code given in the aria-errormessage property uses aria-live=assertive https://www.w3.org/TR/wai-aria-1.2/#aria-errormessage
```<span id="msgID" aria-live="assertive"><span style="visibility:visible">Invalid time:...>```
If aria-live=assertive is "imperative" on an error message, then we should document why.

SOLUTION 1 
Add the sentence after the example:
> ```aria-live="assertive"``` is used in this example instead of ```polite``` because the error message should be read before the user finishes the field and tabs to next field where the previous error message may be confused with the next field's message?

SOLUTION 2
Change the example from ```aria-live=assertive``` to ```polite```
